ruby_on_rails/rails_basics/project_blog_app.md

```diff
@@ -8,7 +8,7 @@ To be honest, you're kind of going into the deep end so don't worry if you don't
 
 <div class="lesson-content__panel" markdown="1">
 
-  1. Do the [The Ruby on Rails Guides: Getting Started](https://guides.rubyonrails.org/getting_started.html) project through section 9.2, from the Rails Guides. It gives a pretty good overview of the common commands you'll use when using Rails. Section 9.3 is causing a few problems (but should be fixed soon). However, if you want to explore additional functionality of deleting or basic authentication, you can do Section 10 and 11.
+  1. Do the [The Ruby on Rails Guides: Getting Started](https://guides.rubyonrails.org/getting_started.html) project through section 9.2. It ties together the Model-View-Controller and gives a pretty good overview of the common commands you'll use when using Rails. You are not expected to complete sections 9.3 and 11.1 on Concerns and Basic Authentication. We have not covered these topics and people often have a difficult time completing these sections because the instructions do not follow the same copy/paste pattern.
```

Comment:
[Not a blocker] Wording is slightly ambiguous around expectations. Do 'through section 9.2', but don't do 9.3 and 11.1. Perhaps simplifying and saying do up to 9.2 would be clearer?

Reply:
I see what you mean by it being slightly ambiguous. I liked your suggestion, but ended up going a step further to clarify. I removed the section numbers to avoid, because the important information is the topics that are covered and the sections numbers made it confusing. I'd like a second thumbs up before I merge, since I drastically changed it.

Reply:
Nice, reads much better I think :+1: